src/graphql.rs

```diff
@@ -432,8 +420,13 @@ where
         Connection::with_additional_fields(has_previous, has_next, additional_fields);
     connection.edges.extend(nodes.into_iter().map(|node| {
         let Ok(node) = node else { unreachable!() };
-        let encoded = encode_cursor(node.unique_key().as_ref());
-        Edge::new(encoded, node.into())
+        let cursor;
+        {
+            let key = node.unique_key();
+            let key = key.as_ref().to_vec();
+            cursor = OpaqueCursor(key);
+        }
```

Comment:
This part has the same issue that I mentioned earlier. Could you correct it?

Reply:
An error occurs when correcting the source as follows.
```
        let key = node.unique_key();
        let key = key.as_ref().to_vec();
        Edge::new(OpaqueCursor(key), node.into())
```
error[E0505]: cannot move out of `node` because it is borrowed
borrow might be used here, when `key` is dropped and runs the destructor for type `<R as UniqueKey>::AsBytes<'_>`

If you have any recommendations, please let me know.

Reply:
I will make the change as below.
```
  let key = node.unique_key().as_ref().to_vec();
  Edge::new(OpaqueCursor(key), node.into())
```